src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,388 @@
+~~~ shell
+cockroach start \
+--certs-dir=certs \
+--advertise-addr=<node1 address> \
+--join=<node1 address>,<node2 address>,<node3 address> \
+--cache=.25 \
+--max-sql-memory=.25 \
+--background \
+--config-profile replication-target
+~~~
+
+Ensure that you follow the [prerequisite deployment guide]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}#step-4-initialize-the-cluster) to initialize your cluster before continuing to set up physical cluster replication.
+
+### Connect to the standby cluster system interface
+
+Connect to your standby cluster's system interface using [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}).
+
+1. To connect to the system interface, pass the `options=-ccluster=system` parameter in the URL:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --url \
+    "postgresql://root@{node IP or hostname}:26257/?options=-ccluster=system&sslmode=verify-full" \
+    --certs-dir "certs"
+    ~~~
+
+    You will find that the prompt will include `system` when you have successfully connected.
+
+1. Add your cluster organization and {{ site.data.products.enterprise }} license to the cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING cluster.organization = 'your organization';
+    ~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING enterprise.license = 'your enterprise license';
+    ~~~
+
+1. Confirm the status of your virtual cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTERS;
+    ~~~
+
+    The output will show the `system` interface, but no `application` virtual cluster:
+
+    ~~~
+    id |   name   | data_state | service_mode
+    ---+----------+------------+---------------
+    1  | system   | ready      | shared
+    2  | template | ready      | none
+    (2 rows)
+    ~~~
+
+    {% include {{ page.version.version }}/physical-replication/template-description.md %}
+
+### Create a user for the standby cluster
+
+If you would like to access the DB Console to observe your replication, you will need to create a user:
+
+1. Create a user:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE USER {your username} WITH LOGIN PASSWORD {'your password'};
+    ~~~
+
+1. To observe the replication activity, your user will need [`admin`]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) privileges:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    GRANT admin TO {your username};
+    ~~~
+
+    Open the DB Console in your web browser: `https://{node IP or hostname}:8080/`, where you will be prompted for these credentials. Refer to [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}) for more detail on tracking relevant metrics for your replication stream.
+
+## Step 3. Copy certificates
```

Comment:
I think some links to here still call this section "Verify certificates".

Reply:
Found, and changed